components/navigation/ui/src/main/resources/XWiki/SearchSuggestConfig.xml

```diff
@@ -289,6 +289,12 @@
       </sectionPriority>
     </class>
     <property>
+      <advancedOnly/>
+    </property>
+    <property>
+      <categoryPriority/>
+    </property>
+    <property>
```

Comment:
Not really, PhenoTips uses an extended version of the `ConfigurableClass`, with more fields than what XWiki has, while this document comes from XWiki. To keep the changes from the official document, we should not include these extra fields.